### PR TITLE
Remove all audio and photogalleries from the navigations

### LINKED ISSUE #5853: Remove all Audio and PhotoGalleries from the WS navs
**Is your feature request related to a problem? Please describe.**
Remove all Audio and PhotoGalleries from the WS navs

**Please check with Jon before doing this ticket! (to keep in sync with work on other platforms)** 

**Describe the solution you'd like**
Remove the following from Navigation

Site | Nav Link | Type
-- | -- | --
gahuza | /gahuza/media/audio | audio
swahili | /swahili/media/audio | audio
swahili | /swahili/media/photogalleries | photogalleries
azeri | /azeri/media/photogalleries | photogalleries
kyrgyz | /kyrgyz/media/audio | audio
afrique | /afrique/media/photogalleries | photogalleries
zhongwen | /zhongwen/simp/media/photogalleries | photogalleries
zhongwen | /zhongwen/simp/media/audio | audio
zhongwen | /zhongwen/trad/media/photogalleries | photogalleries
zhongwen | /zhongwen/trad/media/audio | audio
pashto | /pashto/media/photogalleries | photogalleries
vietnamese | /vietnamese/media/photogalleries | photogalleries
vietnamese | /vietnamese/media/audio | audio
hindi | /hindi/media/photogalleries | photogalleries
burmese | /burmese/media/audio | audio
burmese | /burmese/media/photogalleries | photogalleries
turkce | /turkce/media/photogalleries | photogalleries
urdu | /urdu/media/audio | audio
urdu | /urdu/media/photogalleries | photogalleries
ukrainian | /ukrainian/media/photogalleries | photogalleries
nepali | /nepali/media/photogalleries | photogalleries
pidgin | /pidgin/media/audio | audio
somali | /somali/media/audio | audio
somali | /somali/media/photogalleries | photogalleries
portuguese | /portuguese/media/photogalleries | photogalleries
arabic | /arabic/media/photogalleries | photogalleries
bengali | /bengali/media/audio | audio
bengali | /bengali/media/photogalleries | photogalleries
hausa | /hausa/media/photogalleries | photogalleries
persian | /persian/media/photogalleries | photogalleries

**Describe alternatives you've considered**
None.

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by OlgaLyubin**
Resolves #5853

**Overall change:** 
Removed all Audio and Photogalleries from the WS navigations.

**Code changes:**
- Updated navigation items in service configs.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/lib/config/services/somali.js [hunk 1]
```diff
@@ -250,10 +250,6 @@ export const service = {
         title: 'Barnaamijyada Idaacadda',
         url: '/somali/ka_qayb_gal/programmes',
       },
-      {
-        title: 'Sawirro',
-        url: '/somali/media/photogalleries',
-      },
     ],
```
Comment (DenisHdz) [inline src/app/lib/config/services/somali.js:254]: We also have to remove the audio from the Somali service `/somali/media/audio`

**Review by DenisHdz — CHANGES_REQUESTED** [review]

File: src/app/lib/config/services/somali.js (hunk 1 above)
Comment (OlgaLyubin) [reply src/app/lib/config/services/somali.js:254]: 🙈 Missed that. Fixed now.

**Review by DenisHdz — APPROVED** [review]
LGTM 👍

**Review by thekp — APPROVED** [review]

**Comment by PriyaKR** [comment]
Looks good to me.The audio and photogalleries links are not available on the nav and also the corresponding pages are 404ing.

**Comment by sareh** [comment]
Update: the work for removing these on the PAL application is due to happen on the live environment on Monday 30th March. We should check when the release is happening on that morning and merge in then.

**Comment by sareh** [comment]
The work has gone live for the PAL, so we can merge this in.

**Pull request merged**